lib/std/os/linux/ioctl.zig

```diff
@@ -0,0 +1,56 @@
+const std = @import("../../std.zig");
+
+const bits = switch (std.Target.current.cpu.arch) {
```

Comment:
```suggestion
const bits = switch (@import("builtin").cpu.arch) {
```
`std.Target.current.cpu.arch` is deprecated #9321.